src/libDirectoryService/DirectoryService.cpp

```diff
@@ -472,7 +472,7 @@ void DirectoryService::RejoinAsDS() {
     auto func = [this]() mutable -> void {
       m_mediator.m_lookup->SetSyncType(SyncType::DS_SYNC);
       m_mediator.m_node->CleanVariables();
-      m_mediator.m_node->Install(SyncType::DS_SYNC, true);
+      m_mediator.m_node->Install(SyncType::DS_SYNC);
```

Comment:
I think all our `Install` calls are with `toRetrieveHistory = true`.
Do we still have a case where we might need `toRetrieveHistory = false`?
If not, maybe we can remove this parameter.

Reply:
Probably not, currently the bootstrap command in AWS/local will execute "zilliqa xxx 1 0 0", that is, Node::Install(x, false) is used.